generators/dotnet-templates/Microsoft.BotFramework.CSharp.CoreBot/content/CoreBot.Tests/CoreBot.Tests.csproj

```diff
@@ -21,8 +21,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.Bot.Builder.Testing" Version="4.7.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
```

Comment:
I've gotten mixed feedback on this dependency.  Gabo had me add aspnetcore.app.  so I'd like to get a consistent set of feedback as to which is the appropriate asp.net assemblies to generate for the project.

Reply:
.NET Core 3.1 needs this dependency - they removed it from the core, so we need to add it. It isn't required for 2.2